web/views.py

```
@@ -5181,51 +5181,39 @@ def educational_videos_list(request):
     return render(request, "videos/list.html", context)
 
 
-@login_required
 def upload_educational_video(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `upload_educational_video`

(ANN201)